src/main/java/tc/oc/pgm/commands/ModerationCommands.java

```diff
@@ -2,29 +2,51 @@
 
 import app.ashcon.intake.Command;
 import app.ashcon.intake.CommandException;
+import app.ashcon.intake.parametric.annotation.Switch;
 import app.ashcon.intake.parametric.annotation.Text;
+import com.google.common.base.Strings;
 import com.google.common.cache.Cache;
 import com.google.common.cache.CacheBuilder;
-import java.time.Duration;
+import com.google.common.collect.Lists;
 import java.time.Instant;
+import java.util.List;
 import java.util.UUID;
 import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+import javax.annotation.Nullable;
 import net.md_5.bungee.api.ChatColor;
+import net.md_5.bungee.api.chat.BaseComponent;
 import org.bukkit.Bukkit;
 import org.bukkit.command.CommandSender;
 import org.bukkit.entity.Player;
+import org.joda.time.Duration;
```

Comment:
Don't use joda time, use java time

Reply:
Question regarding joda. It seems that [PeriodFormats](https://github.com/Electroid/PGM/blob/master/src/main/java/tc/oc/util/components/PeriodFormats.java) is using it and I’d like to utilize one of its methods for displaying how much time until a temp ban expires. Any advice?

Reply:
May be out of scope for this PR, but on 1.9+ pgm all the times had been migrated away from joda to java 8  time

Reply:
You should still make the import be java time in new code tho, if a specific place needs joda time make that the fully qualified name. Currently you're just using joda all arround (even on the event) wich you really shouldn't